viewer/viewer-awt/src/main/java/org/icepdf/ri/common/PersistentJCheckBoxMenuItem.java

```diff
@@ -0,0 +1,26 @@
+package org.icepdf.ri.common;
+
+import javax.swing.*;
+import javax.swing.plaf.synth.SynthCheckBoxMenuItemUI;
+
```

Comment:
Nice little class,  can you add small block of documentation for the class indicating what it's purpose is.  It's not super obvious at first glance that this class actually does.  JCheckBox that won't hide when users selects an item?